### PR TITLE
Update 1.12.4

### PR DESCRIPTION
**Pull request by Jrice1317**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,7 +57,7 @@ about:
     command line tools. It is primarily meant to be run inside existing applications. That said, 
     APScheduler does provide some building blocks for you to build a scheduler service or to run a 
     dedicated scheduler process.
-  doc_url: https://apscheduler.readthedocs.io
+  doc_url: https://apscheduler.readthedocs.io/en/3.x/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `https://apscheduler.readthedocs.io` is fine. The redirect is not a critical issue for the `readthedocs` site

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Okay, thanks! So, should I change it back? Or leave it as is, but make note of it for future reference?

Comment (skupr-anaconda) [reply recipe/meta.yaml]: It's better to change it back

**Review by ryanskeith — APPROVED** [review]
LGTM

**Pull request merged**